package/gluon-mesh-batman-adv/files/lib/netifd/proto/gluon_bat0.sh

```diff
@@ -56,8 +92,8 @@ proto_gluon_bat0_setup() {
 proto_gluon_bat0_teardown() {
 	local config="$1"
 
-	ip link del bat0
 	ip link del primary0
+	batctl interface destroy
```

Comment:
Why is the teardown order inverted here?

Reply:
This was probably done by mistake when rebasing. Will fix that.